### PR TITLE
[FOR 5094 remote/control] add unit tests in nacos-core.

### PR DESCRIPTION
**Pull request by brotherlu-xcq**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

for #5094 we add unit tests by directory, module  `nacos-core`  directory `remote/control`.

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: core/src/test/java/com/alibaba/nacos/core/cluster/MemberUtilTest.java [hunk 1]
```diff
@@ -40,7 +40,7 @@
 @RunWith(MockitoJUnitRunner.class)
 public class MemberUtilTest {
     
-    private static final String IP = "1.1.1.1";
+    private static final String IP = "1.1.1.2";
```
Comment (KomachiSion) [inline core/src/test/java/com/alibaba/nacos/core/cluster/MemberUtilTest.java:43]: Why do this change?

**Review by KomachiSion — CHANGES_REQUESTED** [review]

File: core/src/test/java/com/alibaba/nacos/core/cluster/MemberUtilTest.java (hunk 1 above)
Comment (brotherlu-xcq) [reply core/src/test/java/com/alibaba/nacos/core/cluster/MemberUtilTest.java:43]: The unit test in this file cannot pass due, I fix it through changing the IP.

Comment (KomachiSion) [reply core/src/test/java/com/alibaba/nacos/core/cluster/MemberUtilTest.java:43]: But the unit test can pass in CI. Has something in your environment cause cannot pass?

Comment (brotherlu-xcq) [reply core/src/test/java/com/alibaba/nacos/core/cluster/MemberUtilTest.java:43]: My local environment can pass, and I have not change anything about this. It's so confused me,I will check and retry later.

**Comment by brotherlu-xcq** [comment]
this branch is outdate, I will resubmit in a new PR.

**Pull request closed without merging**